tests/libtransmission/torrent-files-test.cc

```diff
@@ -143,24 +143,34 @@ TEST_F(TorrentFilesTest, hasAnyLocalData)
 
 TEST_F(TorrentFilesTest, isSubpathPortable)
 {
-    static auto constexpr Tests = std::array<std::pair<std::string_view, bool>, 15>{ {
+#ifdef _WIN32
+#define PORTABLE_UNIX_NOT_WIN32 false
+#else
+#define PORTABLE_UNIX_NOT_WIN32 true
+#endif
```

Comment:
You don't have to worry about `#undef` this way.

```suggestion
static auto constexpr NotWin32 = TR_IF_WIN32(false, true);
```

Reply:
830bb6ecd3fd974785a6477c94e762fe76c679d9